Release Note Dec 2024 (Apps) (#4907)
SumoLogic/sumologic-documentation: blog-service/2024-12-24-apps.md

---
title: Apps, Solutions, and Collection Integrations - December Release (Observability)
image: https://help.sumologic.com/img/sumo-square.png
keywords:
  - apps
  - releases-notes
hide_table_of_contents: true  
---

import useBaseUrl from '@docusaurus/useBaseUrl';

<a href="https://help.sumologic.com/release-notes-service/rss.xml"><img src={useBaseUrl('img/release-notes/rss-orange2.png')} alt="icon" width="50"/></a>

### New release

We’re excited to announce the release of new Azure Service Bus, Azure API Management, and AWS Serverless Application Models for Sumo Logic.

- **Azure Service Bus**. Azure Service Bus is a fully managed enterprise message broker with message queues and publish-subscribe topics (in a namespace) used for decoupling applications and services from each other. This integration helps in monitoring incoming/outgoing messages, connections, throttled requests, and resource usage of your Service Bus namespace. [Learn more](/docs/integrations/microsoft-azure/azure-service-bus/).

> 

Reviewer: Remove the linespace between the bullet points.

- **Azure API Management**. Azure API Management is a hybrid multicloud platform for managing APIs across different environments. As a platform-as-a-service, it supports the entire API lifecycle and provides near real-time visibility into API status and health, offering essential API Management operations and details for auditing. [Learn more](/docs/integrations/microsoft-azure/azure-api-management/).

> - **AWS Serverless Application Models**. Released the following four SAM’s with Python v3.13 AWS Lambda runtime update with latest libraries.

Reviewer: ```suggestion
- **AWS Serverless Application Models**. Released the following four SAM’s with Python `v3.13` AWS Lambda runtime update with latest libraries.
```

>     - sumologic-securityhub-connector-aws-org - SAM SemanticVersion: 1.0.8.

Reviewer: ```suggestion
    - `sumologic-securityhub-connector-aws-org`. Upgraded the SAM SemanticVersion to `v1.0.8`.
```

Reviewer: Remove the linespace between the bullets and follow the same changes to other points.

    - sumologic-s3-logging-auto-enable - SAM SemanticVersion: 1.0.15.

    - sumologic-aws-cloudtrail-benchmark - SAM SemanticVersion: 1.0.18.

    - sumologic-app-utils - SAM SemanticVersion: 2.0.19.

### Enhancements 

- We're excited to announce the release of the enhancements listed below for the Sumo Logic apps:

>     - **IIS - OpenTelemetry**. Added 10 new monitors for IIS - OpenTelemetry app that will be triggered for different pre-defined conditions. [Learn more](https://help.sumologic.com/docs/integrations/web-servers/iis-10).

Reviewer: ```suggestion
    - **IIS - OpenTelemetry**. Added 10 new monitors that will be triggered for different pre-defined conditions. [Learn more](https://help.sumologic.com/docs/integrations/web-servers/iis-10).
```

Reviewer: Follow the same for other apps listed in this list. This is just to avoid using the app name multiple times.

    - **HAProxy - OpenTelemetry**. Added 5 new monitors for HAProxy - OpenTelemetry app that will be triggered for different pre-defined conditions. [Learn more](https://help.sumologic.com/docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry/#haproxy-alerts).

    - **Memcached - OpenTelemetry**. Added 5 new monitors for Memcached - OpenTelemetry app that will be triggered for different pre-defined conditions. [Learn more](https://help.sumologic.com/docs/integrations/databases/opentelemetry/memcached-opentelemetry/#memcached-alerts).

    - **RabbitMQ - OpenTelemetry**. Added 6 new monitors for RabbitMQ - OpenTelemetry app that will be triggered for different pre-defined conditions. [Learn more](https://help.sumologic.com/docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry/#rabbitmq-alerts).

    - **Cassandra - OpenTelemetry**. Added 9 new monitors for Cassandra - OpenTelemetry app that will be triggered for different pre-defined conditions. [Learn more](https://help.sumologic.com/docs/integrations/databases/opentelemetry/cassandra-opentelemetry/#cassandra-alerts). 

> - **Block Blob**. Updated the Block Blob collection to support collection for Network Flow logs. The Network Security Group (NSG) flow logs will be removed on 30 September 2027. From 30 June 2025, you will no longer be able to generate new NSG flow logs as part of this retirement. For more details refer to Azure [docs](https://learn.microsoft.com/en-us/azure/network-watcher/flow-logs-read?tabs=nsg).

Reviewer: ```suggestion
- **Block Blob**. Updated the Block Blob collection to support collection for Network Flow logs. The Network Security Group (NSG) flow logs will be removed on 30 September 2027. **From 30 June 2025, you will no longer be able to generate new NSG flow logs as part of this retirement**. For more details refer to Azure [docs](https://learn.microsoft.com/en-us/azure/network-watcher/flow-logs-read?tabs=nsg).
```

> - The 28 apps listed below have been updated, and as part of the app installation flow, customers can now create Cloud to Cloud sources:

Reviewer: ```suggestion
- The 28 apps listed below have been updated, and as part of the app installation flow, you can now create Cloud-to-Cloud sources:
```

>     - 1Password

Reviewer: Add the respective app docs link.

    - Okta
    - Abnormal Security
    - Slack
    - CrowdStrike - Falcon Endpoint Protection
    - CrowdStrike Spotlight
    - Cisco Meraki
    - Microsoft Azure AD Inventory
    - Salesforce
    - SentinelOne
    - Microsoft Graph Security
    - Atlassian
    - Netskope
    - Duo Security
    - Tenable
    - Akamai Security Events
    - Sophos
    - Proofpoint On Demand
    - CrowdStrike FDR Host Inventory
    - Qualys VMDR
    - Workday
    - Airtable
    - Box
    - Cato Networks
    - Proofpoint TAP
    - KnowBe4
    - Rapid7
    - Microsoft Graph Azure AD Reporting
    - LastPass

### Bug fixes

- Minor *query* fixes in the following [Next-Gen Apps](/docs/get-started/apps-integrations/#next-gen-apps):
    - Kubernetes
    - EKS Control Plane app
    - Doppel Vision
    
- Minor fixes in the *monitors* in the following [Classic Apps (Legacy)](/docs/get-started/apps-integrations/#classic-apps-legacy):
    - AWS WAF
    - AWS WAF - Cloud Security Monitoring and Analytics

> - Flex app - Minor changes in the variable name.

Reviewer: ```suggestion
- **Flex app**. Minor changes in the variable name.
```

<a href="https://help.sumologic.com/release-notes-service/rss.xml"><img src={useBaseUrl('img/release-notes/rss-orange2.png')} alt="icon" width="50"/></a>

### New release

We’re excited to announce the release of new Azure Service Bus, Azure API Management, Azure Event Grid, and AWS Serverless Application Models for Sumo Logic.

- **Azure Service Bus**. Azure Service Bus is a fully managed enterprise message broker with message queues and publish-subscribe topics (in a namespace) used for decoupling applications and services from each other. This integration helps in monitoring incoming/outgoing messages, connections, throttled requests, and resource usage of your Service Bus namespace. [Learn more](/docs/integrations/microsoft-azure/azure-service-bus/).
- **Azure API Management**. Azure API Management is a hybrid multicloud platform for managing APIs across different environments. As a platform-as-a-service, it supports the entire API lifecycle and provides near real-time visibility into API status and health, offering essential API Management operations and details for auditing. [Learn more](/docs/integrations/microsoft-azure/azure-api-management/).

> - **Azure Event Grid**. Azure Event Grid is a highly scalable, fully managed Pub Sub message distribution service that offers flexible message consumption patterns using the MQTT and HTTP protocols. This integration helps in monitoring data plane requests, delivery failures and publish failures of Event Grid resources - custom topics, system topics and domains. [Learn more](/docs/integrations/microsoft-azure/azure-event-grid/)

Reviewer: ```suggestion
- **Azure Event Grid**. Azure Event Grid is a highly scalable, fully managed Pub Sub message distribution service that offers flexible message consumption patterns using the MQTT and HTTP protocols. This integration helps in monitoring data plane requests, delivery failures and publish failures of Event Grid resources - custom topics, system topics, and domains. [Learn more](/docs/integrations/microsoft-azure/azure-event-grid/).
```
